validation.py

```diff
@@ -4,8 +4,8 @@ def is_on_board(choice, board):
         return any(choice in row for row in board)
 
     @staticmethod
-    def is_in_range(choice, range):
-        return choice in range
+    def is_in_range(choice, board_range):
+        return choice in board_range
 
     @staticmethod
     def is_valid_integer(string):
```

Comment:
are you still using the `is_there_whitespace` method? if not, you can go ahead & delete it to keep your codebase tidy. (& actually, even if you are using it, I'm wondering if it would be simpler to just strip inputs by default?)